client/my-sites/site-settings/manage-connection/index.jsx

```diff
@@ -59,14 +40,8 @@ class ManageConnection extends Component {
 	}
 }
 
-export default connect(
-	( state ) => {
-		const siteId = getSelectedSiteId( state );
-
-		return {
-			siteIsAtomic: isSiteAutomatedTransfer( state, siteId ),
-			siteIsJetpack: isJetpackSite( state, siteId ),
-			siteSlug: getSelectedSiteSlug( state ),
-		};
-	}
-)( localize( ManageConnection ) );
+export default flowRight(
+	( ) => {},
```

Comment:
You'll need to remove that one - it will break the `flowRight` call sequence.

Reply:
Ah yes :)